test/round/RoundImplementation.test.ts

```diff
@@ -1088,18 +1088,27 @@ describe.only("RoundImplementation", function () {
         await initRound(_currentBlockTimestamp);
       });
 
-      const buildNewState = (current: bigint, indexes: number[], statusArray: number[]) => {
-        let newState:bigint = current;
+      const buildStatusRow = (
+        currentRow: bigint,
+        statuses: {index: number, status: ApplicationStatus}[]
+      ) => {
+        if(statuses.length > 128) {
+          throw new Error("Cannot build a status row with more than 128 statuses.");
+        }
+
+        let newRow = currentRow;
+
+        for (let i = 0; i < statuses.length; i++) {
+          const rowIndex = BigInt(statuses[i].index) * 2n;
```

Comment:
since we are building a single row, is this actually the `columnIndex`?

Reply:
good catch